src/main/java/com/google/graphgeckos/dashboard/storage/ParsedBuildbotData.java

```diff
@@ -26,34 +26,35 @@
  * Fields included: 
```

Comment:
ParsedBuildbotData got superseded in #88 . Consider rebasing on that.

Reply:
Given that the file was removed, will @holounic need to integrate these changes onto the <code>Builder</code> class?

Reply:
Yes

Reply:
Pulled origin here. Updated and moved files accordingly.

Reply:
Pulled origin here. Updated and moved files accordingly.